tools/src/commands/PublishProdExpoHomeCommand.ts

```diff
@@ -0,0 +1,222 @@
+
+  const bundlePath = platform === 'ios' ? iosPublishBundlePath : androidPublishBundlePath;
+  await fs.writeFile(path.resolve(projectRoot, bundlePath), await bundleResponse.buffer());
+}
+
+/**
+ * Main action that runs once the command is invoked.
+ */
+async function action(): Promise<void> {
+  console.log('Getting expo-cli state of the current session...');
+  const cliState = await getExpoCliStateAsync();
+  const cliUsername = cliState?.auth?.username;
+  if (cliUsername !== 'exponent') {
+    throw new Error('Must be logged in as `exponent` account to publish');
+  }
+
+  const appJsonFilePath = path.join(EXPO_HOME_PATH, 'app.json');
+
+  const slug = 'home';
+  const owner = 'exponent';
+  const easProjectId = '6b6c6660-df76-11e6-b9b4-59d1587e6774';
+  const easUpdateURL = `https://u.expo.dev/${easProjectId}`;
+
+  const appJsonFile = new JsonFile<AppConfig>(appJsonFilePath);
+  const appJson = await appJsonFile.readAsync();
+
+  if (!appJson.expo.owner) {
+    throw new Error('app.json missing owner');
+  }
+  if (!appJson.expo.extra || !appJson.expo.extra.eas || !appJson.expo.extra.eas.projectId) {
+    throw new Error('app.json missing extra.eas.projectId');
+  }
+  if (!appJson.expo.updates || !appJson.expo.updates.url) {
+    throw new Error('app.json missing updates.url');
+  }
+
+  console.log(`Creating backup of ${chalk.magenta('app.json')} file...`);
+  const appJsonBackup = deepCloneObject<AppConfig>(appJson);
+
+  console.log(`Modifying home's slug to ${chalk.green(slug)}...`);
+  appJson.expo.slug = slug;
+
+  console.log(`Modifying home's owner to ${chalk.green(owner)}...`);
+  appJson.expo.owner = owner;
+
+  console.log(`Modifying home's EAS project ID to ${chalk.green(easProjectId)}...`);
+  appJson.expo.extra.eas.projectId = easProjectId;
+
+  console.log(`Modifying home's update URL to ${chalk.green(easUpdateURL)}...`);
+  appJson.expo.updates.url = easUpdateURL;
+
+  // Save the modified `appJson` to the file so it'll be used as a manifest.
+  await appJsonFile.writeAsync(appJson);
+
+  const createdUpdateGroupId = (
+    await publishAppAsync({ slug, message: `Publish ${appJson.expo.sdkVersion}` })
+  ).createdUpdateGroupId;
+
+  console.log(`Restoring ${chalk.magenta('app.json')} file...`);
+  await appJsonFile.writeAsync(appJsonBackup);
+
+  console.log(`Downloading published manifests and bundles...`);
+  await Promise.all([
+    fetchManifestAndBundleAsync(EXPO_HOME_PATH, easProjectId, createdUpdateGroupId, 'ios'),
+    fetchManifestAndBundleAsync(EXPO_HOME_PATH, easProjectId, createdUpdateGroupId, 'android'),
+  ]);
+
+  console.log(
+    chalk.yellow(
+      `Finished publishing. Remember to commit changes of the embedded manifests and bundles.`
+    )
+  );
+}
+
+export default (program: Command) => {
+  program
+    .command('publish-prod-home')
+    .alias('pph')
+    .description(`Publishes home app for production on EAS Update.`)
```

Comment:
lol, I thought our prettier replaces backticks with single quotes when the string doesn't have any interpolation